src/main/java/org/jenkinsci/test/acceptance/plugins/dashboard_view/read/BreadCrumbs.java

```diff
@@ -18,7 +18,7 @@ public class BreadCrumbs extends PageAreaImpl {
     /**
      * The path to the breadcrumbs list.
      */
-    final By nameCrumb = By.xpath("//ul[@id=\"breadcrumbs\"]");
+    final By nameCrumb = By.xpath("//ol[@id=\"breadcrumbs\"]");
```

Comment:
As I understand the discussion this is for the new breadcrumb - but this will break without the PR and it should be compatable with Jenkins both with and without the upstream PR.

Reply:
```suggestion
    final By nameCrumb = By.xpath("//ol[@id=\"breadcrumbs\"] | //ul[@id=\"breadcrumbs\"]");
```